src/Discord.Net.Commands/Attributes/Preconditions/RequireUserPermissionAttribute.cs

```diff
@@ -11,6 +11,8 @@ public class RequireUserPermissionAttribute : PreconditionAttribute
     {
         public GuildPermission? GuildPermission { get; }
         public ChannelPermission? ChannelPermission { get; }
+        public override string ErrorMessage { get; set; }
+        public string NotAGuild { get; set; }
```

Comment:
I think that `NotAGuildMessage` or `NotAGuildErrorMessage` would be a more fitting name.